src/scripts/main.js

```diff
@@ -1,3 +1,30 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+function success() {
+  document.body.insertAdjacentHTML('beforeend', `
+   <div class="message">Promise was resolved!</div>`);
+}
+
+function error() {
+  setTimeout(() => {
+    document.body.insertAdjacentHTML('beforeend', `
+      <div class="error-message">Promise was rejected!</div>`);
+  }, 3000);
+}
+
+const promise1 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    resolve(success());
+  });
+});
+
+const promise2 = new Promise((resolve, reject) => {
+  logo.addEventListener('click', () => {
+    reject(new Error(error));
+  });
+});
```

Comment:
`promise2` promise should be rejected in 3 seconds.
And this event is not linked to the click on logo or first promise